src/components/CardListBox/CardListBox.tsx

```diff
@@ -0,0 +1,101 @@
+import { ChangeEvent, useEffect, useState } from "react";
+import CardSearchInput from "./CardSearchInput/CardSearchInput.tsx";
+import useGet from "../../hooks/useGet";
+import CardFolderList from "./CardFolderList/CardFolderList.tsx";
+import styles from "./CardListBox.module.scss";
+import CardList from "../CardList/CardList.tsx";
+import { END_POINT } from "../../constants/index.ts";
+import getFormattedLinks from "@/hooks/getFormattedLinks";
+import LinksEmptyCase from "./LinksEmptyCase/LinksEmptyCase.tsx";
+import {
+  FormetLinkProps,
+  LinkProps,
+  UserFolderProps,
+} from "@/constants/index.types";
+
+export default function CardListBox() {
+  const [folders, setFolders] = useState<UserFolderProps[]>([]);
+  const [links, setLinks] = useState<LinkProps[]>([]);
+  const [changeLinksFolder, setChangeLinksFolder] = useState(END_POINT.links);
+  const [linksTitle, setLinksTitle] = useState(`전체`);
```

Comment:
단순히 문자열만 표현한다면 백틱보다 따옴표가 더 좋을 것 같아요!
```suggestion
  const [linksTitle, setLinksTitle] = useState('전체');
```